### DISCUSSION TITLE
Transparent default color

### DISCUSSION THREAD

**Discussion by chrsmlls333** [marcofugaro/three-projected-material#55] [Q&A] [answered]
Great library! This is making reprojection almost seamless.

I'm not sure if this is a support request or bug, but when I skip the `color` property on the material, the default unprojected texture is white. Is there a way to make this default to transparent? Should/could it be transparent if `color` in ProjectedMaterialParameters is undefined?

**Comment by marcofugaro** [comment, marked as answer]
you can make only the projection visible by using the `backgroundOpacity` uniform, in combination with `transparent: true`:

```js
const material = new ProjectedMaterial({
  camera: webgl.camera,
  texture,
  transparent: true,
})
material.uniforms.backgroundOpacity.value = 0

```

**Reply by chrsmlls333** [reply]
Thank you very much for your assistance here! Might be good to make that an option property. Working great!

**Reply by marcofugaro** [reply]
Makes sense, thanks for the suggestion!
